### DISCUSSION TITLE
Setting timeout for getting the response from API

### DISCUSSION THREAD

**Discussion by canob** [grafana/grafana-infinity-datasource#682] [Q&A] [answered]
Hi,
Anyone know if there is a way in Infinity Plugin to configure Client TimeOut?
Because I'm getting this error, "Client.Timeout exceeded while awaiting headers" on Grafana side on a panel, and for example with Loki Datasource, I can configure a timeout for it and prevent that:
![image](https://github.com/yesoreyeram/grafana-infinity-datasource/assets/30271927/e1b95e14-8d7e-4346-a586-96ad6d57b79d)

Thanks in advance for your help!

**Comment by canob** [comment]
Anyone?

**Reply by yesoreyeram** [reply, marked as answer]
In the data source config page, there is a network section. You can configure time-out there.

**Comment by canob** [comment]
Thanks @yesoreyeram , how stupid I am, I don't understand why I didn't see that before.

**Comment by Prathamesh2809** [comment]
@yesoreyeram I tried updating timeout but it is getting overridden somewhere
